curriculum/js1/10.js

```diff
@@ -8,6 +8,10 @@
  */
 
 const solution = (a, b, c) => {
+  a = a*1000;
+  b = b*1000;
+  setTimeout(c, a);
+  setTimeout(c, b);
```

Comment:
if you do this, lets say a is 5 and b is 5, your function will end after 5 seconds. 

Problem wants to run c after 5 seconds, then run c after 5 seconds (10 seconds total)